openapi/sms/v1-review/docs/readme.md

```diff
@@ -0,0 +1,262 @@
+---
+stoplight-id: cz3ydei53qosx
+---
+
+# Emarsys SMS Partner API Documentation
+
+SAP Emarsys empowers marketers around the world to create a truly personalized omnichannel experiences that delivers business outcomes. The Emarsys SMS Partner API allows partners to integrate onto the Emarsys SMS channel. The Partner API uses a RESTful approach.
+
+## API Specification
+
+The API specification has been created using OpenAPI 3.0.3. It is available to be viewed and exported using the links below:
+
+- [Partner Service Specification](../partner-service.yaml)
+- [Partner Callbacks Specification](../partner-callbacks.yaml)
+
+The current functionality supported is:
+
+- [Self-service on-boarding for the client end user](#client-onboarding)
+- [Sending SMS messages triggered by the Emarsys platform](#outbound-messages)
+- [Recording delivery reports of sent messages](#delivery-reports)
+- [Receiving inbound SMS messages](#inbound-messages)
+
+## Glossary
+
+| Term | Definition |
+| ---- | --------- |
+| Client | The client user utilising the Partner service for the SMS channel. |
+| Client ID | The ID used to reference the client user's integration with the Partner service. It is generated when the client user completes the onboarding |
+| Partner Service | The web service operated by the Partner that utilises the Emarsys SMS Partner API |
+
+---
+## Setup
+
+This section describes the steps required to use the SAP Emarsys SMS Partner API.
+
+### Partner Onboarding
+
+To onboard we require:
+
+- Display name of the partner
+- Base URL for the partner service
+- Supported countries (ideally a list of ISO-3 country codes)
```

Comment:
Maybe add a link to the ISO-3 country code list

Reply:
I'm wandering what types of country codes are supported since here it says "ideally".